internal/api/api.go

```diff
@@ -151,6 +152,11 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, userDeviceSvc
 	udOwner.Get("/offers", valuationsController.GetOffers)
 	udOwner.Get("/instant-offer", valuationsController.GetInstantOffer)
 
+	vOwner := v1Auth.Group("/vehicles/:tokenId", deviceMw)
```

Comment:
Instead of the `deviceMw`, the security model here is going to be different - instead using the privilidged JWT. Here is an example of how this works in device-data-api: https://github.com/DIMO-Network/device-data-api/blob/e010a1dc1337c41a66983bb2ad0c0b66605a0200/cmd/device-data-api/api.go#L101

You'll need to bring in the priviligetoken package from shared library. For now use the `privileges.VehicleNonLocationData` as the required permission.